src/builder/server.js

```diff
@@ -204,7 +204,7 @@ export default (webpackConfig, options = {}) => {
     const href = `http://${ devHost }:${ devPort }${ config.baseUrl.pathname }`
     log(`Dev server started on ${ href }`)
     if (config.open) {
-      opn(href)
+      opn(href.replace("0.0.0.0", "localhost"))
```

Comment:
I can't think of any harm when doing this. But this means user can't override default open host.